test/test_baseline_trainers.py

```diff
@@ -140,10 +134,56 @@ def test_cubemap_stiching(test_cfg_path: str, mode: str, camera: str):
             sensor_uuids
         )
     meta_config.freeze()
-    execute_exp(meta_config, mode)
-    # Deinit processes group
-    if torch.distributed.is_initialized():
-        torch.distributed.destroy_process_group()
+    if camera in ["equirect", "fisheye"]:
+        execute_exp(meta_config, mode)
+        # Deinit processes group
+        if torch.distributed.is_initialized():
+            torch.distributed.destroy_process_group()
+
+    elif camera == "cubemap":
+        # 1) Generate an equirect image from cubemap images.
+        # 2) Generate cubemap images from the equirect image.
+        # 3) Compare the input and output cubemap
+        env = habitat.Env(config=config)
+        observations = env.reset()
+        batch = batch_obs([observations])
+        orig_batch = deepcopy(batch)
+
+        #  ProjectionTransformer
+        obs_trans_to_eq = baseline_registry.get_obs_transformer(
+            "CubeMap2Equirect"
+        )
+        cube2equirect = obs_trans_to_eq(sensor_uuids, (256, 512))
+        obs_trans_to_cube = baseline_registry.get_obs_transformer(
+            "Equirect2CubeMap"
+        )
+        equirect2cube = obs_trans_to_cube(
+            cube2equirect.target_uuids, (256, 256)
+        )
+
+        # Cubemap to Equirect to Cubemap
+        batch_eq = cube2equirect(batch)
+        batch_cube = equirect2cube(batch_eq)
+
+        # Extract input and output cubemap
+        output_cube = batch_cube[cube2equirect.target_uuids[0]]
+        input_cube = torch.cat([orig_batch[key] for key in sensor_uuids])
+
+        # Apply blur to absorb difference (blur, etc.) caused by conversion
+        if sensor_type == "RGB":
+            output_cube = output_cube.float() / 255
+            input_cube = input_cube.float() / 255
+        output_cube = output_cube.permute((0, 3, 1, 2))  # NHWC => NCHW
+        input_cube = input_cube.permute((0, 3, 1, 2))  # NHWC => NCHW
+        apply_blur = torch.nn.AvgPool2d(5, 3, 2)
+        output_cube = apply_blur(output_cube)
+        input_cube = apply_blur(input_cube)
+
+        # Calculate the difference
+        diff = torch.abs(output_cube - input_cube)
```

Comment:
This is fine I suppose, but L1 isn't the best distance. Maybe best to compare with SSIM? Otherwise, this diff looks pretty good.

Reply:
Sure, I can use SSIM.

Reply:
Hold on. Are you sure you want to use SSIM? Currently, there is no dependency on `scikit-image` in habitat-lab. But I wanna use it for [SSIM](https://scikit-image.org/docs/dev/auto_examples/transform/plot_ssim.html).
I'm not sure if it is a good idea to include `scikit-image` just for testing. Also, only the difference caused by conversion should be some blur and a little misalignment because of `align_corners=True`, so I think L1 after downsampling images is enough in this case.

Reply:
Wow, does OpenCV really not have any easily accessible Python SSIM bindings??
Neither does PyTorch??
Tensorflow does, but that's overkill. 

I guess L1 is fine in this case then.